deploy/metrics-apiservice.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.metrics
+spec:
+  insecureSkipTLSVerify: true
+  group: metrics
```

Comment:
> We should keep metrics as the group name for backwards compat, but the eventual trend is to move to qualified group names, AFAIK

Why not do that now, while you're in alpha and can change things? I don't understand